binr/rax2/rax2.c

```diff
@@ -29,39 +29,39 @@ static int format_output(char mode, const char *s) {
 	}
 	switch (mode) {
 	case 'I':
-		printf ("%" PFMT64d "\n", n);
+		printf ("%" PFMT64d "%c", n, newline);
```

Comment:
whats the point of showing a number without newline? is this useful in any situation?